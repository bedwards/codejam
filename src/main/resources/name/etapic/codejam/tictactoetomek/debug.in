1
T.O.
....
....
.X..

