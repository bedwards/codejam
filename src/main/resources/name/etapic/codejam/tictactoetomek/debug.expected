Case #1: Game has not completed
